README.md

```diff
@@ -33,12 +45,60 @@ The license returned from operator is a simple dictionary of form;
 
 Where data is the named attributes in the [spdx template license specification](https://spdx.github.io/spdx-spec/v2.2.2/license-matching-guidelines-and-templates/) for sections named "var" and uses the names as defined in the template.
 
+### Fuzzy Match
+
+The above code is for exact match, which only match the license text exactly with cache fingerprint fetch from
+[spdx license](https://github.com/spdx/license-list-data/tree/main/json). Sometime license text may not be
+exactly match, and you still want to detect the license from the text. For this case, you can use fuzzy match
+
+An extra dependency is required for fuzzy matching. Install it before using fuzzy match:
+
+```bash
+pip install spdx_matcher[fuzzy]
+```
+
+And the fuzzy code as below:
+
+```python
+import spdx_matcher
+
+with open("LICENSE.txt") as myf:
+    license_text = myf.read()
+
+fuzzy_result = spdx_matcher.fuzzy_license_text(license_text)
+```
+
+The result of fuzzy match is different from exact match. The result is a list of dictionary, each dictionary
+contains the license and the percentage of match result.
```

Comment:
One more thing I have to mention, the result format is not the same as fuzzy match result, the init reason is fuzzy match has a score attribute, if we separate it into two different key like `licenses` and `execpetions`, we would not get the orders easier. So I combine them into one single list.

But the result format of fuzzy match and `analyse_license_text ` is not same, I am not sure whether is a good idea or not